use-xdg-dir.patch

```diff
@@ -0,0 +1,50 @@
+From d601d2ee1216cd5f09e90c7c627f39326e8de936 Mon Sep 17 00:00:00 2001
+From: Qv2ray-Bot <59914293+Qv2ray-Bot@users.noreply.github.com>
+Date: Fri, 24 Jan 2020 21:11:11 +0800
+Subject: [PATCH] add: added flatpak config path support
+
+---
+ Qv2ray.pro   | 1 +
+ src/main.cpp | 9 +++++++++
+ 2 files changed, 10 insertions(+)
+
+diff --git a/Qv2ray.pro b/Qv2ray.pro
+index 321fe60..6bca50c 100644
+--- a/Qv2ray.pro
++++ b/Qv2ray.pro
+@@ -323,6 +323,7 @@ with_metainfo {
+     appdataXml.files += ./assets/qv2ray.metainfo.xml
+     appdataXml.path = $$PREFIX/share/metainfo/
+     INSTALLS += appdataXml
++    DEFINES += WITH_FLATHUB_CONFIG_PATH
+ }
+ 
+ message(" ")
+diff --git a/src/main.cpp b/src/main.cpp
+index 6346ab9..1e19d1d 100644
+--- a/src/main.cpp
++++ b/src/main.cpp
+@@ -93,6 +93,10 @@ bool initialiseQv2ray()
+     QStringList configFilePaths;
+     configFilePaths << QDir::homePath() + "/.qv2ray" QV2RAY_CONFIG_DIR_SUFFIX;
+     configFilePaths << QDir::homePath() + "/.config/qv2ray" QV2RAY_CONFIG_DIR_SUFFIX;
++#ifdef WITH_FLATHUB_CONFIG_PATH
++    LOG(MODULE_INIT, "---> Adding flakpak config path support.")
++    configFilePaths << QDir::homePath() + "/.var/app/com.github.Qv2ray/config/qv2ray" QV2RAY_CONFIG_DIR_SUFFIX;
```

Comment:
Any reason for not using QStandardPaths instead of hardcoding the path?

Reply:
I am working on it

Reply:
Fixed. @bilelmoussaoui